### PR TITLE
[terra-avatar] Add way to convey patient is deceased to assistive technologies

### PR DESCRIPTION
**Pull request by cschetan77**
### Summary
Terra Avatar currently relies on color alone to convey a deceased patient avatar. This PR contains changes that adds a way for assistive technologies to convey deceased patient for non sighted users.

Closes #

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by JessieRandle** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Please include a CHANGELOG entry for each changed package this PR. Looks like a CHANGELOG is missing for: 

 - terra-avatar</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f79a5a531c655b543e85db73b1621a5c8f1584ba
</p>

File: packages/terra-avatar/src/common/AvatarUtils.jsx [hunk 1]
```diff
@@ -68,13 +68,21 @@ const generateImagePlaceholder = (avatarParams) => {
   return <span className={avatarIconClassNames} role="img" aria-label={alt} alt={alt} aria-hidden={isAriaHidden} />;
 };
 
+/**
+ * Appends deceased text based on alt
+ */
+
+const appendDeceased = (alt, initials) => (alt === '' ? `${initials}, Deceased` : `${alt}, Deceased`);
```
Comment (saket2403) [inline packages/terra-avatar/src/common/AvatarUtils.jsx:75]:
```suggestion
const appendDeceased = (alt, initials) => (alt === '' ? `${initials}, intl.formatMessage({ id: 'Terra.avatar.deceased' })` : `${alt}, intl.formatMessage({ id: 'Terra.avatar.deceased' })`);
```

File: packages/terra-avatar/translations/en.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "Terra.avatar.deceased": "Deceased"
+}
```
Comment (saket2403) [inline packages/terra-avatar/translations/en.json:3]: You need to log a translations JIRA for this entry to be translated into other terra supported languages as well.

Comment (cschetan77) [reply packages/terra-avatar/translations/en.json:3]: Logged, here - https://jira.cerner.com/browse/TRANS-14113

File: packages/terra-avatar/src/common/AvatarUtils.jsx (hunk 1 above)
Comment (cschetan77) [reply packages/terra-avatar/src/common/AvatarUtils.jsx]: Updated.

**Comment by cschetan77** [comment]
@saket2403 @udaymattam @sugan2416 Please review this PR again.

**Review by saket2403 — APPROVED** [review]

**Review by sugan2416 — APPROVED** [review]

File: packages/terra-avatar/translations/en-GB.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "Terra.avatar.deceased": "Deceased"
+}
```
Comment (supreethmr) [inline packages/terra-avatar/translations/en-GB.json:3]: add a new line after this for consistency with other translations file

**Review by supreethmr — APPROVED** [review]

**Pull request merged**